src/components/Sidebar.vue

```diff
@@ -10,9 +12,20 @@ const spacesStore = useSpacesStore();
     <router-link :to="{ name: 'home' }" class="h-[72px] block">
       <IH-stop class="inline-block my-4 w-[32px] h-[32px] text-skin-link" />
     </router-link>
-    <draggable v-model="spacesStore.starredSpacesIds" :item-key="i => i" class="space-y-3 p-2">
+    <draggable
+      v-model="spacesStore.starredSpacesIds"
+      :delay="100"
+      :delay-on-touch-only="true"
+      :touch-start-threshold="35"
+      :item-key="i => i"
+      class="space-y-3 p-2"
+    >
       <template #item="{ element }">
-        <router-link :to="{ name: 'overview', params: { id: element } }" class="block">
+        <router-link
+          :to="{ name: 'overview', params: { id: element } }"
+          class="block"
+          @click="uiStore.sidebarOpen = false"
```

Comment:
```suggestion
```

Do we need this change?

We do this:
https://github.com/snapshot-labs/sx-ui/blob/ecea212b7969c2467d43060fb6bd61680523aa51/src/App.vue#L39-L41

Reply:
We need to handle a click on already selected space, so route watcher doesn't handle that case.